### PR TITLE
New wake up scheduler pluging

### PR DESCRIPTION
**Pull request by Damian-AU**
New wake up scheduler pluging

### CODE REVIEW

**Comment by Mimoja** [comment]
Did not yet have the time to test, might need a day or two, thank you so much !

**Comment by Damian-AU** [comment]
Could I get feedback on this?  feeling discourage by the silence, please advise if that is the intention?

**Comment by Mimoja** [comment]
@Damian-AU Sorry, this was merged 6 days ago by me and i forgot to push

**Review by Mimoja — COMMENTED** [review]
the .DS_Store files should not be commited.

File: de1plus/plugins/DPx_Scheduler/plugin.tcl [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+        set DPx_shed {}
+        foreach k [lsort -dictionary [array names item]] {
+            set v $item($k)
+            append DPx_shed [subst {[list $k] [list $v]\n}]
+        }
+        write_file $fn $DPx_shed
+    }
+
+    proc minutes_digits {} {
+        if {$::plugins::DPx_Scheduler::minutes >= 50 && $::plugins::DPx_Scheduler::minutes < 60} {
```
Comment (Mimoja) [inline de1plus/plugins/DPx_Scheduler/plugin.tcl:336]: You can use "variable" here, which shortens these a bit

like this:
https://github.com/decentespresso/de1app/blob/main/de1plus/plugins/visualizer_upload/plugin.tcl#L231

or this
https://github.com/decentespresso/de1app/blob/4b6a2959032aca7962ac2b7565c71389d4a8a4fc/de1plus/history_viewer.tcl#L77

**Review by Mimoja — COMMENTED** [review]
Rest looked good

**Comment by ebengoechea** [comment]
I join the suggestion. Using "variable" on namespace procs avoids having to type fully qualified names, you then treat the variables as local variables, plus it is slightly more efficient.

I would also suggest to avoid introducing any new global variables in an extension. Under normal circunstances all its data should be kept in namespace variables. You already removed several globals, but still have ::DPx_scheduler_times and ::DPx_scheduler_minutes.

**Pull request closed without merging**